clang/include/clang/CIR/Dialect/IR/CIRDataLayout.h

```diff
@@ -12,7 +12,6 @@
 #ifndef LLVM_CLANG_LIB_CIR_CIRDATALAYOUT_H
 #define LLVM_CLANG_LIB_CIR_CIRDATALAYOUT_H
```

Comment:
You need to change the header guards too: `LLVM_CLANG_CIR_DIALECT_IR_CIRDATALAYOUT_H`